bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/classic/BulkReadClassicApi.java

```diff
@@ -44,6 +44,12 @@
     this.tableId = tableId;
   }
 
+  @Override
+  public void close() {
+    isClosed = true;
+    sendOutstanding();
```

Comment:
Do we need to check
`if (isClosed) return` ? 

Should the order be:

sendOutstanding();
isClosed = true; ?

Reply:
Practically I dont think it matters in this case because add() & close() can be called only from a single thread. However, logically I would want to prevent addition of elements first and then flush the buffered ones